### DISCUSSION TITLE
selective reupload

### DISCUSSION THREAD

**Discussion by ntn888** [demanuel/NewsUP#94] [Q&A] [answered]
Hi,

Is there an option to reupload only what's missing again from the temp folder? I've looked through the wiki but couldn't find it. Perhaps i can modify the before-upload script to do this?

I had to cancel the default integrety check, as the propogation to the second nttp wasnt complete and it was trying to upload the *entire* content again.

----

btw the '-check' option confirms the nzb is not complete, 99%.

**Comment by demanuel** [comment, marked as answer]
> Is there an option to reupload only what's missing again from the temp folder? 

Unfortunately it's not possible. Although if you still have the files in the temp folder, and have a list of files missing you can upload them manually without any compression and without par'ing them. You'll have then to merge both nzb (this might be labourious).

> modify the before-upload script

Using the before_upload script wouldn't make much sense to use in this case. The problem is after the upload.

> I had to cancel the default integrety check, as the propogation to the second nttp wasnt complete and it was trying to upload the _entire_ content again.

You can increase the waiting time before checking the headers.

> btw the '-check' option confirms the nzb is not complete, 99%.

If you used par files with more than 1% redudancy, the upload is still good ;-)

**Comment by ntn888** [comment]
Oh okay thanks for your response.

--If you used par files with more than 1% redudancy, the upload is still good ;-)--

Good to know. That's what I thought too... But for some reason attempt with NZBGet didnt' suceed.

**Reply by demanuel** [reply]
Did the nzbget downloaded the pars? Theoretically it should download the pars if the nzb isn't complete... But unfortunately i cannot help you with nzbget.

**Reply by ntn888** [reply]
No worries. I'll look into it. :)
